libindy/tests/pairwise.rs

```diff
@@ -281,31 +281,31 @@ mod high_cases {
             assert_ne!(pairwise_info_with_metadata, pairwise_info_without_metadata);
             assert_eq!(format!(r#"{{"my_did":"{}"}}"#, my_did), pairwise_info_without_metadata);
 
-            utils::tear_down_with_wallet(wallet_handle);
+            utils::tear_down_with_wallet(wallet_handle, "indy_set_pairwise_metadata_works_for_reset", &wallet_config);
         }
 
         #[test]
         fn indy_set_pairwise_metadata_works_for_not_created_pairwise() {
-            let wallet_handle = utils::setup_with_wallet();
+            let (wallet_handle, wallet_config) = utils::setup_with_wallet("indy_set_pairwise_metadata_works_for_not_created_pairwise");
```

Comment:
`setup_with_wallet` can generate and return random string (as wallet name) instead of explicit passing.

Reply:
It is easier to see this way what did not get cleaned up.